Makefile

```diff
@@ -75,3 +99,4 @@ down-pg_translit:
 clean: down-redis down-pg_translit down-pg
 	find . -name __pycache__ -exec rm -rf {} +
 	find . -name "*.pyc" -exec rm -rf {} +
+	rm compose-env/*.env
```

Comment:
You're right. I didn't consider that. Even worse for production. (do we back up our `*.env` files there?)

I've removed that line from the recipe: 67377736f9580dfc8071608e805d9e2862023a43